src/FixerDefinition.php

```diff
@@ -38,7 +38,7 @@ public function __construct(
         array $codeSamples,
         $configurationDescription,
         array $defaultConfiguration = null,
-        $riskyDescription
+        $riskyDescription = null
```

Comment:
please revert, I don't want to use default values anywhere here. So I want to enforce user to think about that parameter and try to fill it, instead of just skipping as "optional"

`array $foo = null` is there because I cannot use `?array $foo` yet.